zigcc-armv7-unknown-linux-gnueabihf.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env sh
+/home/runner/.cargo/bin/cargo-zigbuild zig cc -- -target arm-linux-gnueabihf -mcpu=generic+v7a+vfp3-d32+thumb2-neon -g $@
```

Comment:
What is this script for? And why does it need to be in the project root?

Reply:
that was a failed experiment. removed now